device-types/Cisco/C8300-6T

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: Cisco
+model: Catalyst 8300-6T
+slug: c8300-12x
+part_number: C8300-6T
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-micro-b
+  - name: usbc
+    type: usb-c
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 650
```

Comment:
Power supplies are modular, these should be setup as modules with each power supply as a module-type:

PWR-CC1-250WAC
PWR-CC1-500WAC
PWR-CC1-400WDC
PWR-CC1-400WHV